src/components/ReportActionItem/MoneyRequestPreview/MoneyRequestPreviewContent.tsx

```diff
@@ -186,7 +186,7 @@ function MoneyRequestPreviewContent({
     };
 
     const showContextMenu = (event: GestureResponderEvent) => {
-        if (!shouldDisplayContextMenu) {
+        if (!shouldDisplayContextMenu || !reportID) {
```

Comment:
Is this change needed, I don't how it is related to eslint problem, and why can't we leave it as it is?